README.rst

```diff
@@ -25,7 +25,7 @@ Dependencies
 
 The following dependencies will be necessary for IOData to build properly,
 
-* Python >= 2.7, >= 3.5: http://www.python.org/
+* Python >= 3.5: http://www.python.org/
```

Comment:
Thanks for catching this. We actually need 3.6 and higher because we're using f-strings. 

```suggestion
* Python >= 3.6: http://www.python.org/
```